src/qt/veil/settings/settingsfaq02.cpp

```diff
@@ -5,37 +5,7 @@ SettingsFaq02::SettingsFaq02(QWidget *parent) :
     QWidget(parent),
     ui(new Ui::SettingsFaq02)
 {
-    ui->setupUi(this);
-
```

Comment:
I am not sure if we wan't to delete the ui->setupUI(this). This can cause issues, where pointers are set to null, and will cause the client to crash in certain cases.